src/template/ext/top/configure/RULES_JAVA

```diff
@@ -1,13 +1,8 @@
-#*************************************************************************
-# Copyright (c) 2002 The University of Chicago, as Operator of Argonne
-#     National Laboratory.
 # Copyright (c) 2002 The Regents of the University of California, as
-#     Operator of Los Alamos National Laboratory.
+# Copyright (c) 2002 The University of Chicago, as Operator of Argonne
+# SPDX-FileCopyrightText: 2000 Argonne National Laboratory
+#
 # SPDX-License-Identifier: EPICS
-# EPICS Base is distributed subject to a Software License Agreement found
-# in file LICENSE that is included with this distribution. 
-#*************************************************************************
-
```

Comment:
I should probably delete this file, but until I do I think your tool mangled the existing copyright header a bit, please fix.

Reply:
Uh, I totally missed this. The tool I used tries to preserve the original copyright notices, but apparently can't handle multiline notices. Are you ok with just replacing all the notices with SPDX tags? They may not have been correct in the first place. I will try to trace the history of the file again, but it seems to be older than the notices state, I had previously traced its origins to the year 2000.

Reply:
Ah, I found the reason. Janet added a bunch of copyright notices in 2002. The file was originally [added by Marty](65d9c939bc0bb5e03cf040e71927ae83e44061c6). So, 2000 appears to be the correct date.